### PR TITLE
Create kotlin-cheatsheet.md

### PR DESCRIPTION
**Pull request by rahuldas-404**
## Change Summary

Provide summary of changes with issue number if any.

## Checklist

**If you haven't fulfilled the below requirements or even delete the entire checklist, your PR won't be reviewed and will be closed without notice.**

### General

- [x] This Pull Request is all my own work. (You'll be blacklisted if you are caught for plagiarism.)
- [x] I've read [CONTRIBUTING.md](https://github.com/aryankashyap7/CheatSheets-for-Developers/blob/main/docs/CONTRIBUTING.md)
- [x] I've made some valid changes to the CheatSheet, and they are not just minor changes.

### Changes

- [x] I've read and followed the [Solution Template](https://github.com/aryankashyap7/CheatSheets-for-Developers/blob/main/docs/CONTRIBUTING.md#submission-template)

### CODE REVIEW

File: CheatSheets/kotlin-cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+# PHP Cheatsheet for Developers
```
Comment (aryankashyap7) [inline CheatSheets/kotlin-cheatsheet.md:1]: Its a Kotlin Cheatsheet

File: CheatSheets/kotlin-cheatsheet.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+## Table of Contents
```
Comment (aryankashyap7) [inline CheatSheets/kotlin-cheatsheet.md:3]: The hyperlinks for Basic, If Else, While loop, and Collections aren't working properly!

**Review by aryankashyap7 — CHANGES_REQUESTED** [review]
Update your PR with all these new changes!

File: CheatSheets/kotlin-cheatsheet.md (hunk 1 above)
Comment (rahuldas-404) [reply CheatSheets/kotlin-cheatsheet.md:1]: ok, I'll change it!!

File: CheatSheets/kotlin-cheatsheet.md (hunk 2 above)
Comment (rahuldas-404) [reply CheatSheets/kotlin-cheatsheet.md:3]: ok, I'll rectify it!

**Comment by aryankashyap7** [comment]
This looks great 🍁

**Pull request merged**